doc/srv6/srv6_vpn.md

```diff
@@ -0,0 +1,284 @@
+# SRv6 VPN HLD
+
+# Overview
+
+Alibaba is deploying SRv6 VPN with SONiC based white box router at its edge of the network. The Alibaba's SRv6 VPN use case has been described in muliple conferences. The basic idea follows RFC 8986, which we use SRv6 as a network programming framework for applications.  VPN information is used as service anchor points for identifying service instances in the service plane. SRv6 policy could be used to provide paths to let applications reach service instances. The following diagram provides a summary on what Alibaba would like to achieve via SRv6 VPN.
+
+![image](images/1PGN.png)
+
+To help promoting SRv6's adopting in the industry, Alibaba team decides to upstream its deployed SRv6 VPN solution to SONiC community via 202305 release. We expect more contributors would join us to complete SRv6 VPN related features.
+
+# Design
+
+The SONiC SRv6 VPN deployable solution  requires changes from the following three communities.
+
+## FRR
+
+Besides recent FRR changes for SRv6 VPN contributed by Cisco team, Alibaba team needs to bring in the following FRR changes for a deployable solution. 
+
+1.  VRF routes leaking and performance tuning
+    
+2.  BGP advertise delay
+    
+3.  Conditional advertisement
+    
+4.  SRv6 Policy handling
+    
+
+FRR changes would be commited to FRR community.  SONiC 202305 release would pull in the corresponding  version of FRR release.
+
+The minimal FRR version for SRv6 VPN is 8.4, which you could get SRv6 VPN PoC functionalities. FRR version 8.5 or 9.0 would provide Alibaba deployment feature parity.
```

Comment:
not sure if SONiC HLD should talk about parity with a specific customer of vendor. Should community HLD require to mention anything on parity with Alibaba?

Reply:
I want to give community a confidence on these are deployed codes in Alibaba cloud, not a PoC toy.